src/components/modals/SurveyModal.tsx

```diff
@@ -6,47 +6,77 @@ import Animated, { Easing, useAnimatedStyle, useSharedValue, withTiming } from '
-  { label: lstrings.survey_opt_youtube, selected: false },
-  { label: lstrings.survey_opt_search_engine, selected: false },
-  { label: lstrings.survey_opt_x_twitter, selected: false },
-  { label: lstrings.survey_opt_in_person_event, selected: false },
-  { label: lstrings.survey_opt_personal_referral, selected: false },
-  { label: lstrings.survey_opt_article, selected: false }
+interface SurveyCategory {
+  catKey: string
+  label: string
+  selected: boolean
+}
+
+const SURVEY_CATS: SurveyCategory[] = [
+  { catKey: 'ads', label: lstrings.survey_opt_article, selected: false },
+  { catKey: 'appStore', label: lstrings.survey_opt_app_store, selected: false },
+  { catKey: 'article', label: lstrings.survey_opt_article, selected: false },
+  { catKey: 'inPerson', label: lstrings.survey_opt_in_person_event, selected: false },
+  { catKey: 'referral', label: lstrings.survey_opt_personal_referral, selected: false },
+  { catKey: 'search', label: lstrings.survey_opt_search_engine, selected: false },
+  { catKey: 'social', label: lstrings.survey_opt_youtube, selected: false }
 ]
 
+interface LocaleSubcategory {
+  [langKey: string]: string
+}
+
+// TODO: Update info server with finalized type
+const SURVEY_SUBCATS_TEMP: { [catKey: string]: LocaleSubcategory[] } = {
+  ads: [{ en: 'Social media ad (YouTube, X, Instagram, etc.)' }, { en: 'Website display/banner ad' }, { en: 'Search engine ad' }, { en: 'Podcast ad' }],
+  appStore: [{ en: 'Featured in the app store' }, { en: 'Search result in the app store' }],
+  article: [{ en: 'Crypto-specific blog' }, { en: 'General tech blog' }, { en: 'News outlet' }, { en: 'Influencer blog' }],
+  inPerson: [{ en: 'Conference/trade show' }, { en: 'Meetup' }, { en: 'Hackathon' }, { en: 'Crypto Canal', fr: 'Le Canal Crypto' }],
+  referral: [{ en: 'Friend or family recommendation' }, { en: 'Online forum discussion' }, { en: 'Community group (e.g., Discord, Telegram)' }],
+  search: [{ en: 'Google' }, { en: 'Bing' }, { en: 'DuckDuckGo' }, { en: 'Brave' }],
+  social: [
+    { en: 'X (Twitter)' },
+    { en: 'YouTube' },
+    { en: 'Instagram' },
+    { en: 'LinkedIn' },
+    { en: 'Reddit' },
+    { en: 'TikTok' },
+    { en: 'Facebook' },
+    { en: 'Cris Cyborg' }
+  ]
+}
+
 export const SurveyModal = (props: { bridge: AirshipBridge<void> }) => {
   const { bridge } = props
   const dispatch = useDispatch()
   const theme = useTheme()
   const styles = getStyles(theme)
 
   // Randomize response options, excluding "Other"
-  const randomizedSurveyOpts = [
-    ...shuffleArray([...SURVEY_OPTS, ...(infoServerData.rollup?.installSurvey?.surveyOptions ?? []).map(option => ({ label: option, selected: false }))]),
-    { label: lstrings.survey_opt_other_specify, selected: false }
-  ]
-  const [options, setOptions] = useState(randomizedSurveyOpts)
+  const [options, setOptions] = useState([...shuffleArray(SURVEY_CATS), { catKey: 'other', label: lstrings.survey_opt_other_specify, selected: false }])
```

Comment:
Same thing. You are going to re-caculate this initial value on every render, so do `useState(() => [...`, to make it only run once at mount.